src/styles/antd-overrides/button.scss

```diff
@@ -1,17 +1,18 @@
 .ant-btn {
   background: transparent;
-  @apply border-haze-400 dark:border-slate-300;
+  @apply border-bluebs-400 dark:border-slate-300;
   @apply text-over-action-secondary;
   @apply font-medium;
-  border-radius: 1px;
+  border-radius: 8px;
```

Comment:
**suggestion** lets apply a tailwind class here. if there isnt one for 8px, lets make one: https://tailwindcss.com/docs/border-radius#customizing-your-theme